pyseir/icu/utils.py

```diff
@@ -28,10 +28,13 @@ def calculate_case_based_weights() -> dict:
     last_month_cum_cases.name = "summed_cases"
 
     df = last_month_cum_cases.reset_index().dropna()
-    df["state_fips"] = df["fips"].str.slice(0, 2)  # Apply State Labels to Groupby
+    # Example location_id value = 'iso1:us#iso2:us-ak#fips:02013'
+    df["state_location_id"] = df["location_id"].str.split("#").str[1]  # 'iso2:us-ak'
+    df["fips_location_id"] = df["location_id"].str.split("#").str[-1].str.split(":").str[-1]
```

Comment:
Is ` _add_fips_if_missing` at all handy? https://github.com/covid-projections/covid-data-model/blob/master/libs/datasets/timeseries.py#L328-L331

Reply:
Updated to use that helper function instead of rolling my own parser.